packages/remark-jargon/package.json

```diff
@@ -35,7 +35,7 @@
     "remark-html": "^13.0.1"
   },
   "dependencies": {
-    "unist-util-visit": "^2.0.3"
+    "unist-util-visit": "^4.1.0"
```

Comment:
this happened because I ran `yarn reconfigure` and this was the version set in the dependencies file